docs/spec/channel.md

```diff
@@ -391,6 +384,23 @@ Channels SHOULD expose a variety of metrics, including, but not limited to:
 Metrics SHOULD be enabled by default, with a configuration parameter included to
 disable them if desired.
 
+The Channel MUST recognize and pass through all tracing information from sender
+to subscribers using [W3C Tracecontext](https://w3c.github.io/trace-context/),
+although internally it MAY use another mechanism(s) to propagate the tracing
+information. The Channel SHOULD sample and write traces to the location
+specified in
+[`config-tracing`](https://github.com/knative/eventing/blob/master/config/config-tracing.yaml).
+
+Spans emitted by the Broker SHOULD follow the
```

Comment:
Any reason why the spans emitted by the channel shold not emit these same attributes?

Reply:
It seems copy-paste typo of https://github.com/knative/eventing/blob/master/docs/spec/broker.md#observability
```suggestion
Spans emitted by the Channel SHOULD follow the
```

Reply:
Whoops, thanks for catching that 🤭